pandas/core/generic.py

```diff
@@ -2740,17 +2740,20 @@ class      (index) object 'bird' 'bird' 'mammal' 'mammal'
     def to_latex(self, buf=None, columns=None, col_space=None, header=True,
                  index=True, na_rep='NaN', formatters=None, float_format=None,
                  sparsify=None, index_names=True, bold_rows=False,
-                 column_format=None, longtable=None, escape=None,
-                 encoding=None, decimal='.', multicolumn=None,
-                 multicolumn_format=None, multirow=None):
+                 column_format=None, longtable=None, caption=None,
```

Comment:
Typically new kwargs go at the end, in case users pass kwargs as positional arguments, which would be crazy here.

Reply:
can you add these options at the end